frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -79,14 +81,50 @@ class OverrideForkPointDialog(
             getString(
               "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.inferred"
             ),
-            radioButtonComment
+            branch.forkPoint?.shortHash ?: "cannot resolve commit hash"
           ),
           OverrideOption.INFERRED
         )
           .comment(
-            thisRowComment
+            escapeHtml4(branch.forkPoint?.shortMessage) ?: "cannot resolve commit message"
```

Comment:
For `null` input param, what does `escapeHtml4` return? also `null` (no NPE)?

Reply:
null - I ve check that before. I should have mention it here in the comment that I am aware of what I am doing

Reply:
Okay if so then pls mark both param and return type of `escapeHtml4` as `@PolyNull` in stubs, see https://checkerframework.org/manual/#handling-warnings-step2

Reply:
done